tai64/src/lib.rs

```diff
@@ -435,3 +448,154 @@ mod tests {
         }
     }
 }
+
+// Serde Impls
```

Comment:
This crate already has `serde` support so these are unnecessary and also unrelated to `zeroize` support.

Reply:
I was wondering where this code for serde was coming from, I cloned the repo again and still the code seems to reappear. Turns out vscode is caching changes that I did long ago and inserting them into the new code automatically every time I open the editor. I will resolve this issue first